keentools/tracker/calc_timer.py

```diff
@@ -205,7 +212,12 @@ def get_settings(cls) -> Any:
 
     @classmethod
     def get_loader(cls) -> Any:
```

Comment:
It appears that the overrides of this function in derived classes can be removed, as they are identical

Reply:
It will be removed totally in next PR